### DISCUSSION TITLE
Why is draft/preset_load.h unidirectional?

### DISCUSSION THREAD

**Discussion by Arakula** [free-audio/clap#287] [General]
Shouldn't there be an equivalent preset_save functionality?
If not, is that really only usable for factory preset files and/or preset files created completely outside the host's control?
Just wondering ...

**Comment by abique** [comment]
So when saving the preset, the plugin may ask for metadata to the user, and has preferred location maybe.

The only thing we could do would be: please save your preset to "this" file.
But we don't know the file name, we don't know the extension either, and we don't know what metadata the user would possibly want to save.
